solarforecastarbiter/validation/tasks.py

```diff
@@ -392,7 +392,7 @@ def _daily_validation(session, observation, start, end, base_url):
                 observation.name, start, end)
     observation_values = session.get_observation_values(
         observation.observation_id, start, end)
-    value_series = observation_values['value']
+    value_series = observation_values['value'].astype(float)
```

Comment:
Why are these `astype(float)` calls needed? To be super secure? My understanding of the PR is that it should prevent the need for this by making appropriate changes to `io.api` or `io.utils`.

Reply:
Yes, just to be super secure since one wouldn't need to use data from the API in these functions